src/Symfony/Component/Notifier/Bridge/FakeSms/FakeSmsEmailTransport.php

```diff
@@ -64,10 +64,12 @@ protected function doSend(MessageInterface $message): SentMessage
             throw new UnsupportedMessageTypeException(__CLASS__, SmsMessage::class, $message);
         }
 
+        \assert($message instanceof SmsMessage); // to keep psalm happy
```

Comment:
we don't use `assert()` in Symfony

Reply:
I've grepped for usage of `assert` within Symfony, before using it, and it used in some places. Anyway, it was an unsuccessful attempt to satisfy psalm. I've just removed it.